src/test/java/impl/ParseDataServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.List;
+import model.FruitTransaction;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ParseDataServiceImplTest {
+    private static final FruitTransaction[] TestsTransaction = new FruitTransaction[] {
```

Comment:
```suggestion
    private static final FruitTransaction[] testTransactions = new FruitTransaction[] {
```
don't capitalize var's name